docs/README.md

```diff
@@ -613,7 +619,12 @@ Booster applications provide you with a GraphQL API out of the box. You send com
 _mutations_ and get read models data using _queries_ or _subscriptions_. 
 
 In this section, we will be sending requests by hand using the online tool [Postwoman](https://postwoman.io/graphql), 
-which is free and includes great support for GraphQL. However, you can use any HTTP client you want.
+which is free and includes great support for GraphQL. However, you can use any HTTP client you want. In Postwoman
+online tool, you should enter the following URL as Endpoint:
+
+```text
+https://<some random number>.execute-api.us-east-1.amazonaws.com/production/graphql
```

Comment:
This is specific to AWS. I think we should not use anything that is provider-specific if we can avoid it. 
We already talked about the `httpURL` output, so we can rewrite this like:
```suggestion
<httpURL>/graphql
```